### ISSUE TITLE
Named Chapters

### ISSUE THREAD

**Issue by vincentscode** [audiamus/BookLibConnect#49]
Hi, I noticed that while https://github.com/audiamus/AaxAudioConverter supports named chapters https://github.com/audiamus/BookLibConnect seems to just number chapters even though all the necessary information should be provided in `content_metadata_xxx.json`.
Is there any technical reason for this or would this be easy to port over?
Thanks in advance!

**Comment by audiamus** [comment]
It's completely depending on the book, i.e. the editor who prepared the book for Audible. Chapter information is downloaded when BookLibConnect requests the license. And the json response will have the chapter info as provided by the server. Whatever is found there, BookLibConnect will pack it into the content_meta_ASIN.json

**Comment by vincentscode** [comment]
Yes. But shouldn't BookLibConnect be able to apply those Chapter Markers from the json to the output file (or potentially even split the output into multiple files based on that)?

**Comment by audiamus** [comment]
Well, there is a clear separation of functionality. As you will have read in the readme, the long-term plan is to fully integrate Book Lib Connect into AAX Audio Converter. And AAX Audio Converter has the splitting functions, together with the silence detection and chapter mark adjustment, if the chapter marks are inaccurate and cut the the chapter title mid-word. Book Lib Connect concentrates on the download and, for the time being, preparing the downloaded book for AAX Audio Converter, emulating the output that the official Audible Win10 app produced.

**Comment by vincentscode** [comment]
I see! Unfortunately I cannot use AAX Audio Converter at the moment as I cannot get my access token. It would seem logical to me to be able to use AAX Audio Converter without an access token on files already decrypted with BookLibConnect but that does not seem to be possible. Is there any reason for that?

**Comment by vincentscode** [comment]
Oh, my bad! I just jumped over to this repo when the README directed me here instead of reading the next line. Thanks for your help! :)
